tests/integration_tests/test_stable_diffusion.py

```diff
@@ -0,0 +1,104 @@
+import base64
+from typing import Dict
+from unittest.mock import patch
+
+import pytest
+
+from aidial_adapter_bedrock.aws_client_config import (
+    AWSClientConfigFactory,
+    UpstreamConfig,
+)
+from aidial_adapter_bedrock.deployments import ChatCompletionDeployment
+from tests.integration_tests.constants import BLUE_PNG_PICTURE
+from tests.utils.mock_storage import MockFileStorage
+from tests.utils.openai import (
+    user,
+    user_with_attachment_data,
+    user_with_image_content_part,
+)
+
+
+def get_upstream_headers(region: str) -> Dict[str, str]:
+    return {
+        AWSClientConfigFactory.UPSTREAM_CONFIG_HEADER_NAME: UpstreamConfig(
+            region=region
+        ).json()
+    }
+
+
+def is_base64(s: str) -> bool:
+    try:
+        base64.b64decode(s)
+        return True
+    except Exception:
+        return False
+
+
+@pytest.fixture
+def mock_v3_storage():
+    storage = MockFileStorage.create()
+    with patch(
+        "aidial_adapter_bedrock.llm.model.stabililty.v3.create_file_storage",
+        return_value=storage,
+    ):
+        yield storage
+        storage.cleanup()
+
+
+@pytest.mark.parametrize(
+    "deployment, region, messages",
+    [
+        (
+            ChatCompletionDeployment.STABILITY_STABLE_DIFFUSION_3_LARGE_V1,
+            "us-west-2",
+            [user("generate image of a dog")],
+        ),
+        (
+            ChatCompletionDeployment.STABILITY_STABLE_DIFFUSION_3_LARGE_V1,
+            "us-west-2",
+            [
+                user_with_image_content_part(
+                    "generate image of a dog", BLUE_PNG_PICTURE
+                )
+            ],
+        ),
+        (
+            ChatCompletionDeployment.STABILITY_STABLE_DIFFUSION_3_LARGE_V1,
```

Comment:
Maybe add a meaningful image-to-image test? because I have no idea what it will produce for this blue pixel. You could save an asset with png image and mock file storage to read a file directly from the disk.

Or you could use https://httpbin.org/image/jpeg with a prompt "replace the background with jungles" or something like this,

Reply:
Done, used Claude V3.5 to verify output from both `text-to-image` and `image-to-image`

Added static dog image from this Amazon [tutorial](https://github.com/aws-samples/amazon-bedrock-workshop/blob/main/04_Image_and_Multimodal/bedrock-titan-image-generator-v2.ipynb)

Reply:
With image-to-image test was:

There is dog and basket with food on image
I've added prompt "Dog with red flowers in basket nearby"

And after that I've asked Claude "Is there dog and red flowers on the image? Answer only YES or NO."